api/app/src/command/medical/document/check-doc-queries.ts

```diff
@@ -70,12 +74,19 @@ export async function getPatientsToUpdate(
   patientIds?: string[]
 ): Promise<Patient[]> {
   const query = getQuery(propertyName, maxTime, patientIds);
+
   const patientsWithDocQueriesInProgress = (await PatientModel.sequelize?.query(query, {
     type: QueryTypes.SELECT,
     mapToModel: true,
     instance: new PatientModel(),
   })) as Patient[];
 
+  // 'updatedAt' is not being set by sequelize, so we need to set it manually
+  patientsWithDocQueriesInProgress.forEach(patient => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    if (!patient.updatedAt) patient.updatedAt = (patient as any)["updated_at"];
+  });
+
```

Comment:
OOC - any idea *why*?

Looked into it a bit and the field should be getting set, "in theory".

Reply:
Yeah, did a couple tests, nothing... the rest is set correctly with the options below... maybe because that's inherited from the base class?

```ts
    mapToModel: true,
    instance: new PatientModel(),
```